### ISSUE TITLE
Skill Required wrong

### ISSUE THREAD

**Issue by BSS101** [pyfa-org/Pyfa#2464]
See attachment, that is not correct according to Eve....? when I double click the all 5 , nothing happens 

### Release or development git branch? Please note the release version or commit hash:
newest version

### Operating system and version (eg: Windows 10, OS X 10.9, OS X 10.11, Ubuntu 16.10):
Monterey

### Other relevant information:
<img width="395" alt="Miner" src="https://user-images.githubusercontent.com/64458000/185745666-c6760e70-1209-4723-b916-3b237fc4517f.png">

**Comment by DarkFenX** [comment]
This is the tooltip which should be shown when you hover your mouse over skillbook in top-right (when it's red, i.e. some skills are missing), and it shows missing skills for whole fit, not just this module. It is being shown there indicates that there is weird something with how pyfa's UI toolkit detects where your mouse is.

When you say "high slots not working", what do you mean? Clicking them does not do anything (e.g. changing selection)?

**Comment by BSS101** [comment]
Exactly. I have a Procurer in fittings with empty slots, when I double click or drag a Civilian Miner from Market, nothing happens. If I go down a tab and double click on a Mining Laser Upgrade I, the lower slot is filled. I've tried my hand at it and it seems to be a problem associated with the ship. It happens to all Mining barges and Exhumers...

**Comment by DarkFenX** [comment]
I am unsure how it is done on macos, but can you see interface scaling factor somewhere?
